redash/handlers/authentication.py

```diff
@@ -46,15 +46,24 @@ def render_token_login_page(template, org_slug, token, invite):
             ),
             400,
         )
-    except (SignatureExpired, BadSignature):
-        logger.exception("Failed to verify invite token: %s, org=%s", token, org_slug)
+    except SignatureExpired:
+        logger.exception("Token signature has expired. Token: %s, org=%s", token, org_slug)
         return (
             render_template(
                 "error.html",
                 error_message="Your invite link has expired. Please ask for a new one.",
             ),
             400,
         )
+    except BadSignature:
+        logger.exception("Bad signature for the invite token: %s, org=%s", token, org_slug)
```

Comment:
These code is duplicated. Maybe u should set `error_message = `, and use same render_template.

Reply:
Like this style:
`
try:
    ...
except Excepte1:
 ...
 error_message = ...
except Excepte2:
...
error_message = ...

return (
render_template = ... # at there u can use error_message = error_message
)
`

Maybe this style is a simple code.

Reply:
you mean using `error_message` as a flag?

```python
error_message = ""

try: 
    ...
except Excepte1: 
    ...
    error_message = ... 
except Excepte2: 
    ...
    error_message = ...

if error_message:
    return (
        render_template("error.html", error_message=error_message),
        400,
    )
```

Reply:
done :rocket: 

personally i value readability more than duplicated code but this is fine too